docs/sources/enterprise/saml/about-saml.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'About SAML authentication in Grafana'
+menuTitle: 'About SAML authentication'
+description: 'SAML authentication'
+keywords: ['grafana', 'saml', 'documentation', 'saml-auth']
+aliases: ['/docs/grafana/latest/auth/saml/']
+weight: 20
+---
+
+# SAML authentication
```

Comment:
The h1 should match the title, so in this case, the h1 should be "About SAML authentication in Grafana"